ADD_NEW_DATASET.md

```diff
@@ -1,357 +0,0 @@
-# How to add one (or several) new datasets to 🤗 Datasets

```

Comment:
I think some links are still pointing to this file. Maybe we can just redirect to the Hub or the doc page on how to add a dataset ?

Reply:
Normally, I changed all the links to this file...

Reply:
I have readded the file with just the links to our docs do add datasets to the Hub: 6c58778